logstreams/src/main/java/io/camunda/zeebe/logstreams/impl/log/LogEntryDescriptor.java

```diff
@@ -103,6 +104,18 @@ public final class LogEntryDescriptor {
     METADATA_OFFSET = offset;
```

Comment:
:x: The offset calculations need to be adjusted, `METADATA_LENGTH_OFFSET` still shifts only by one short.

Reply:
We then shift by a SHORT for the unused part, so it's still correct, but confusing :smile:

Reply:
I've updated it to make it less confusing